.github/workflows/test.yml

```diff
@@ -53,6 +53,8 @@ jobs:
 
       - name: Check Request
         id: check_request
+        env:
+          BOT_NAME: ${{ secrets.BOT_NAME }}
```

Comment:
PR title says BOT_TOKEN. Do you want to set BOT_NAME or BOT_TOKEN in the env ?

Reply:
Thx, was trying to be too quick :-)